adafruit_sharpmemorydisplay.py

```diff
@@ -62,7 +65,15 @@ class SharpMemoryDisplay(adafruit_framebuf.FrameBuffer):
 
     # pylint: disable=too-many-instance-attributes,abstract-method
 
-    def __init__(self, spi, scs_pin, width, height, *, baudrate=2000000):
+    def __init__(
+        self,
+        spi: SPIDevice,
```

Comment:
for `spi` I think it's the type `busio.SPI` 

For the code it can be either:
```
import busio
# or 
from busio import SPI
```

And then the type:
```
spi: busio.SPI,
# or 
spi: SPI,
```